migration/data/seedTokens.ts

```diff
@@ -1881,6 +1881,198 @@ const seedTokens: ITokenData[] = [
+    address: '0x1e4a5963abfd975d8c9021ce480b42188849d41d',
+    coingeckoId: 'tether',
+    isGivbackEligible: true,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'Dai Stablecoin',
+    symbol: 'DAI',
+    decimals: 18,
+    address: '0xc5015b9d9161dca7e18e32f6f25c4ad850731fd4',
+    coingeckoId: 'dai',
+    isGivbackEligible: true,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'PancakeSwap Token',
+    symbol: 'CAKE',
+    decimals: 18,
+    address: '0x0D1E753a25eBda689453309112904807625bEFBe',
+    coingeckoId: 'pancakeswap-token',
+    isGivbackEligible: true,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'Rocket Pool ETH',
+    symbol: 'rETH',
+    decimals: 18,
+    address: '0xb23c20efce6e24acca0cef9b7b7aa196b84ec942',
+    coingeckoId: 'rocket-pool-eth',
+    isGivbackEligible: true,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'Frax',
+    symbol: 'FRAX',
+    decimals: 18,
+    address: '0xFf8544feD5379D9ffa8D47a74cE6b91e632AC44D',
+    coingeckoId: 'frax',
+    isGivbackEligible: true,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'Staked MATIC',
+    symbol: 'stMATIC',
+    decimals: 18,
+    address: '0x83b874c1e09d316059d929da402dcb1a98e92082',
+    coingeckoId: 'lido-staked-matic',
+    isGivbackEligible: true,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'Balancer',
+    symbol: 'BAL',
+    decimals: 18,
+    address: '0x120ef59b80774f02211563834d8e3b72cb1649d6',
+    coingeckoId: 'balancer',
+    isGivbackEligible: true,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'Curve DAO Token',
+    symbol: 'CRV',
+    decimals: 18,
+    address: '0x3d5320821bfca19fb0b5428f2c79d63bd5246f89',
+    coingeckoId: 'curve-dao-token',
+    isGivbackEligible: false,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'Gravita Debt Token',
+    symbol: 'GRAI',
+    decimals: 18,
+    address: '0xca68ad4ee5c96871ec6c6dac2f714a8437a3fe66',
+    coingeckoId: 'grai',
+    isGivbackEligible: false,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'ChainLink Token',
+    symbol: 'LINK',
+    decimals: 18,
+    address: '0x4b16e4752711a7abec32799c976f3cefc0111f2b',
+    coingeckoId: 'chainlink',
+    isGivbackEligible: false,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'EURA aka agEUR',
+    symbol: 'EURA',
+    decimals: 18,
+    address: '0xa61beb4a3d02decb01039e378237032b351125b4',
+    coingeckoId: 'ageur',
+    isGivbackEligible: false,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'Aave Token',
+    symbol: 'AAVE',
+    decimals: 18,
+    address: '0xa61beb4a3d02decb01039e378237032b351125b4',
+    coingeckoId: 'aave',
+    isGivbackEligible: false,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'Wrapped BTC',
+    symbol: 'WBTC',
+    decimals: 8,
+    address: '0xea034fb02eb1808c2cc3adbc15f447b93cbe08e1',
+    coingeckoId: '',
```

Comment:
Here you forgot the id I guess "wrapped-bitcoin"

Reply:
Thanks I added that